src/network/serverpackethandler.cpp

```diff
@@ -471,6 +471,12 @@ void Server::process_PlayerPos(RemotePlayer *player, PlayerSAO *playersao,
 	v3f position((f32)ps.X / 100.0f, (f32)ps.Y / 100.0f, (f32)ps.Z / 100.0f);
 	v3f speed((f32)ss.X / 100.0f, (f32)ss.Y / 100.0f, (f32)ss.Z / 100.0f);
 
+	const v3s16 old_node_position = floatToInt(playersao->getBasePosition(), BS);
+	const v3s16 new_node_position = floatToInt(position, BS);
+
+	if(old_node_position != new_node_position)
```

Comment:
wrong code style

Reply:
I don't understand, can you please clarify?

Reply:
Space after 'if'.

Reply:
Also the order is unintuitive, it's better as:
`if (new != old)`